packages/components/input/src/index.vue

```diff
@@ -434,17 +437,20 @@ export default defineComponent({
       inputOrTextarea.value.select()
     }
 
-    const handleCompositionStart = () => {
+    const handleCompositionStart = (event) => {
```

Comment:
**Nitpick:** would you mind annotating the `event`'s type?

Reply:
done.